qsutil.h

```diff
@@ -2,7 +2,11 @@
 #define QSUTIL_H
 
 extern void log1();
-extern void log2();
+extern void qslog2();
```

Comment:
If you touch that line anyway, then please also provide the arguments in the prototype.

And when we already have _deprecated_ (which _I_ forgot about), then define log2() as an inline function that is annotated as deprecated, so it will spit warnings on use.